src/sentry/static/sentry/app/views/groupDetails/actions.jsx

```diff
@@ -418,10 +419,36 @@ export default React.createClass({
     this.onUpdate({isBookmarked: !this.getGroup().isBookmarked});
   },
 
+  onDiscard() {
+    let group = this.getGroup();
+    let project = this.getProject();
+    let org = this.getOrganization();
```

Comment:
Musing: at some point, I'd like us to get rid of these mixin helpers and just go through `this.context`.